templates/golang.sh.erb

```diff
@@ -1,3 +1,5 @@
 GOPATH="<%= @workspace %>"
 PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
-export GOPATH PATH
+GO15VENDOREXPERIMENT=1
```

Comment:
Not sure if it's worth making this configurable seeing as it is just an experiment